Dashboard/src/components/common/Panel.js

```diff
@@ -2,22 +2,50 @@ import React from 'react';
 import refresh from '../assets/refresh.svg';
 import Spinner from './Spinner';
 
-const Panel = (props) => {
-  return (
-    <div className="panel-body">
-      <div className="panel-title">
-        <h3 className="panel-title-text">
-          {props.title}
-        </h3>
-        <div className="refresh-container" onClick={props.refreshData}>
-          <img className="refresh" src={refresh} alt={"refresh icon"} />
+export default class Panel extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      content_visible: true,
+      width: window.innerWidth,
+    };
+    this.toggleDropdown = this.toggleDropdown.bind(this);
+  }
+
+  render() {
```

Comment:
I think general convention is to have render() function at the end of your React component

Reply:
fixed